dev/rclone.ts

```diff
@@ -0,0 +1,262 @@
+export const completion: Fig.Spec = {
+  name: "rclone",
+  description: "The Swiss army knife of cloud storage",
+  subcommands: [
+    {
+      name: "about",
+      description: "Get quota information from the remote.",
+      args: {},
+    },
+    {
+      name: "authorize",
+      description: "Remote authorization.",
+      args: {},
+    },
+    {
+      name: "backend",
+      description: "Run a backend specific command.",
+      args: {},
+    },
+    {
+      name: "cat",
+      description: "Concatenates any files and sends them to stdout.",
+      args: {},
+    },
+    {
+      name: "check",
+      description: "Checks the files in the source and destination match.",
+      args: {},
+    },
+    {
+      name: "cleanup",
+      description: "Clean up the remote if possible.",
+      args: {},
+    },
+    {
+      name: "config",
+      description: "Enter an interactive configuration session.",
+    },
+    {
+      name: "copy",
+      description: "Copy files from source to dest, skipping already copied.",
+      args: {},
+    },
+    {
+      name: "copyto",
+      description: "Copy files from source to dest, skipping already copied.",
+      args: {},
+    },
+    {
+      name: "copyurl",
+      description: "Copy url content to dest.",
+      args: {},
+    },
+    {
+      name: "cryptcheck",
+      description: "Cryptcheck checks the integrity of a crypted remote.",
+      args: {},
+    },
+    {
+      name: "cryptdecode",
+      description: "Cryptdecode returns unencrypted file names.",
+      args: {},
+    },
+    {
+      name: "dedupe",
+      description:
+        "Interactively find duplicate filenames and delete/rename them.",
+      args: {},
+    },
+    {
+      name: "delete",
+      description: "Remove the files in path.",
+      args: {},
+    },
+    {
+      name: "deletefile",
+      description: "Remove a single file from remote.",
+      args: {},
+    },
+    {
+      name: "genautocomplete",
+      description: "Output completion script for a given shell.",
+      args: {},
+    },
+    {
+      name: "gendocs",
+      description: "Output markdown docs for rclone to the directory supplied.",
+      args: {},
+    },
+    {
+      name: "hashsum",
+      description: "Produces a hashsum file for all the objects in the path.",
+      args: {},
```

Comment:
This is also fine, but you could do the following

```suggestion
      args: {
        suggestions: ["MD5", "SHA-1", "DropboxHash", "QuickXorHash"]
      },
```
to make it even more awesome 🎉